integration-tests/src/tests/standard_cases/mod.rs

```diff
@@ -402,14 +412,20 @@ pub fn trying_to_create_implicit_account(node: impl Node) {
         )
         .unwrap();
 
-    let cost = fee_helper.create_account_transfer_full_key_cost_fail_on_create_account()
-        + fee_helper.gas_to_balance(
-            fee_helper.cfg().fee(ActionCosts::create_account).send_fee(false)
-                + fee_helper
-                    .cfg()
-                    .fee(near_primitives::config::ActionCosts::add_full_access_key)
-                    .send_fee(false),
-        );
+    let cost = match receiver_id.get_account_type() {
+        AccountType::NearImplicitAccount => {
+            fee_helper.create_account_transfer_full_key_cost_fail_on_create_account()
+                + fee_helper.gas_to_balance(
+                    fee_helper.cfg().fee(ActionCosts::create_account).send_fee(false)
+                        + fee_helper
+                            .cfg()
+                            .fee(near_primitives::config::ActionCosts::add_full_access_key)
+                            .send_fee(false),
+                )
```

Comment:
nit: Can you multiline this by extracting the intermediate values to helper variables? VS code has a feature to extract to a variable if you highlight the code and right click on it.